src/Microsoft.DotNet.Darc/src/Darc/Options/AddBuildToChannelCommandLineOptions.cs

```diff
@@ -40,6 +40,9 @@ internal class AddBuildToChannelCommandLineOptions : CommandLineOptions
         [Option("sdl-validation-continue-on-error", HelpText = "Ignore SDL validation errors.")]
         public string SDLValidationContinueOnError { get; set; }
 
+        [Option("bypass-promotion-pipeline", HelpText = "Add the build to the channel but don't run the build promotion pipeline.")]
```

Comment:
I wonder if we should reword this a bit, I don't know if this is enough for someone to understand what this switch actually does. How about "Add the build to the channel without publishing assets to new feeds?"

Reply:
Makes sense. I'm going to change it.